lib/wings/model_transformer.rb

```diff
@@ -201,8 +205,9 @@ def self.run(obj, keys)
 
     private
 
-      def minted_id
-        ::Noid::Rails.config.minter_class.new.mint
+      def mint_id
```

Comment:
I know this is a private method but do you mind putting in some comments to explain what it is doing and what it is supposed to be returning?  I find line 210 hard to read and understand.

Reply:
does the updated version, without the control-flow `&&`, read more clearly?

Reply:
I think this is helpful.  I guess `assign_id` is the source of my confusion and that carries over into this method.

Reply:
yeah, it's unfortunate that `#assign_id` can return `nil`. especially since `#id=` errors when `nil` is passed. but there we are :|